cli/tests/integration_tests.rs

```diff
@@ -222,24 +222,6 @@ itest!(_029_eval {
   output: "029_eval.out",
 });
 
-itest!(_030_xeval {
-  args: "xeval console.log($.toUpperCase())",
```

Comment:
aren't the tests still valuable?

Reply:
I agree - even though deno_std has unit tests, I think the fact that xeval is a subcommand justifies having these integration tests.

Reply:
+1 for keeping the test -- this also ensures we do not have unexpected problems e.g. API change caused issues when upgrading std deps

Reply:
👍 I left out 043, one for each arg is sufficient here.